mods/gen4/random-teams.js

```diff
@@ -61,6 +61,14 @@ class RandomGen4Teams extends RandomGen5Teams {
 			'Adaptability':1, 'Hustle':1, 'Iron Fist':1, 'Skill Link':1,
 		};
 
+		// Give recovery moves priority over certain other defensive status moves
+		let recoveryMoves = {
+			'healorder':1, 'milkdrink':1, 'moonlight':1, 'morningsun':1, 'recover':1, 'rest':1, 'roost':1, 'slackoff':1, 'softboiled':1, 'synthesis':1, 'wish':1,
```

Comment:
Yeah because it requires another move, Sleep Talk. Rejecting a move in the hope of getting _two_ moves is risky. Also I don't think you even check for Sleep Talk when you're rejecting.